tests/unit/dataactvalidator/test_scripts_load_sf133.py

```diff
@@ -11,15 +11,15 @@
 FINGERPRINT_COLS = [
     'availability_type_code', 'sub_account_code', 'allocation_transfer_agency', 'fiscal_year',
     'beginning_period_of_availa', 'ending_period_of_availabil', 'main_account_code', 'agency_identifier',
-    'period', 'created_at', 'updated_at', 'tas']
+    'period', 'created_at', 'updated_at', 'tas', 'display_tas']
 
 
 def test_fill_blank_sf133_lines_types():
     """Validate that floats aren't downgraded to ints in the pivot_table
     function (that'd be a regression)."""
     data = pd.DataFrame(
         # We'll only pay attention to two of these fields
-        [[1440, 3041046.31] + list('ABCDEFGHIJKL')], columns=['line', 'amount'] + FINGERPRINT_COLS
+        [[1440, 3041046.31] + list('ABCDEFGHIJKLL')], columns=['line', 'amount'] + FINGERPRINT_COLS
```

Comment:
Just checking, why did we add a duplicate letter at the end here? (and duplicate numbers lower down)

Reply:
This test was setup weirdly with the `FINGERPRINT` setup. Long story short, the L here (and the duplicate numbers lower down) represent the TAS string for each row. So to make it account for Display TAS, I simply copied the same letter/number to be consistent.